sql/sql_admin.cc

```diff
@@ -1014,6 +1014,9 @@ static bool mysql_admin_table(THD* thd, TABLE_LIST* tables,
 
     if (table->table)
     {
+      const bool skip_flush=
```

Comment:
Just a sugestion - maybe different name for this variable? skip_flush is quite vage while we have clear reason why we skip flush. Maybe is_table_being_analized ?

Reply:
Perhaps, but definitely not is_table_being_analyzed, because of the second part after &&: "(table->table->file->ha_table_flags() & HA_ONLINE_ANALYZE)"